packages/android_alarm_manager/example/pubspec.yaml

```diff
@@ -4,8 +4,7 @@ description: Demonstrates how to use the android_alarm_manager plugin.
 dependencies:
   flutter:
     sdk: flutter
-  android_alarm_manager:
-    path: ../
+  android_alarm_manager: {path: ../} # ^x.y.z in your application
```

Comment:
nit: I'd make the comment more verbose so it's easier to understand why it's a path dependency here, maybe something like:

```
The example app is bundled with the plugin so we use a path dependency on the parent directory to use the current plugin's version. When depending on this package from a real application you should use : plugin_name: ^x.y.z
See https://dart.dev/tools/pub/dependencies#version-constraints for more information.
```

Reply:
Good point, that's better than using one-line :)